client/components/RowB.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+// eslint-disable-next-line import/extensions
+import Stars from './Stars.jsx';
+
+function RowB({ stars, description }) {
+  return (
+    <div className="row_b">
+      <Stars starCount={stars} />
```

Comment:
The work of the Stars component may be able to be done here, see component for comment..

Reply:
Yea i was trying to keep it pure since it renders description as well. but i could be wrong on that though.

Reply:
Gotcha, not sure what the best practice would be in this case

Reply:
yeah ill look into it but sure your right def a lot of refactoring in my components.